try {
    hello();
} catch (error) {
    console.log(error);
}

try {
    anhother();
} catch {
    console.log('esto es un error');
}
